src/vs/editor/contrib/codeAction/common/types.ts

```diff
@@ -193,6 +193,7 @@ export class CodeActionItem {
 	constructor(
 		public readonly action: languages.CodeAction,
 		public readonly provider: languages.CodeActionProvider | undefined,
+		public toMark?: Boolean,
```

Comment:
[nit] Use `boolean` instead. `Boolean` is the boolean object which you rarely want in TS